src/biokbase/narrative/tests/narrative_mock/mockclients.py

```diff
@@ -516,10 +517,10 @@ def __init__(self, token=None):
         pass
 
     def check_workspace_jobs(self, params):
-        raise ServerError("JSONRPCError", -32000, "Job lookup failed.")
+        raise generate_ee2_error("check_workspace_jobs")
 
     def check_job(self, params):
-        raise ServerError("JSONRPCError", -32000, "Check job failed")
+        raise generate_ee2_error("check_job")
```

Comment:
These are both ee2 errors, so fix them to return the correct error type (they were returning workspace server errors).